packages/components/psammead-amp-geo/src/index.jsx

```diff
@@ -60,11 +60,13 @@ const ampGeoData = {
 };
 
 const jsonInlinedScript = data => (
-  <script
-    type="application/json"
-    /* eslint-disable-next-line react/no-danger */
-    dangerouslySetInnerHTML={{ __html: JSON.stringify(data) }}
-  />
+  <>
```

Comment:
I don't think you'll need to update `psammead-amp-geo` - just `packages/utilities/psammead-assets/src/amp-boilerplate.jsx`